SwiftyBeaverTests/BaseDestinationTests.swift

```diff
@@ -343,9 +343,14 @@ class BaseDestinationTests: XCTestCase {
             path: "/world/ViewController.swift",
             function: "myFunc",
             message: "Hello World"))
+        
+        XCTAssertFalse(destination.shouldLevelBeLogged(.Verbose,
+            path: "/world/ViewController.swift",
+            function: "myFunc",
+            message: "Hello World"))
 
         // not in filter but matching global minLevel
-        XCTAssertTrue(destination.shouldLevelBeLogged(.Info,
+        XCTAssertFalse(destination.shouldLevelBeLogged(.Info,
```

Comment:
Its false since the excludes keeps the log from happening. But this does make it a redundant test with the next test below. I have updated the test to reflect better coverage and be more clear in the comments.